### PR TITLE
CVSL-1906 add date formatting

### PR DESCRIPTION
**Pull request by gurps-singh**
This PR is to add some date formatting following the retrieval of the results from the backend caseload. Following some testing locally, found that the dates were formatted incorrectly. This is so we match exactly how the caseload looked before.

### CODE REVIEW

File: server/views/pages/approve/cases.njk [hunk 1]
```diff
@@ -184,7 +184,7 @@
             {
                 text: "Release date",
                 attributes: {
-                    "aria-sort": "ascending" if approvalNeededView else "none"
```
Comment (gurps-singh) [inline server/views/pages/approve/cases.njk:187]: **note:** ‏The release date is now sorted from the backend so this is not needed.

File: server/routes/approvingLicences/handlers/approvalCases.ts [hunk 2]
```diff
@@ -18,9 +20,17 @@ export default class ApprovalCaseRoutes {
...
-    const cases = approvalNeededView
-      ? await this.approverCaseloadService.getApprovalNeeded(user, prisonCaseloadToDisplay, searchTerm)
-      : await this.approverCaseloadService.getRecentlyApproved(user, prisonCaseloadToDisplay, searchTerm)
+    const cases = (
+      approvalNeededView
+        ? await this.approverCaseloadService.getApprovalNeeded(user, prisonCaseloadToDisplay, searchTerm)
+        : await this.approverCaseloadService.getRecentlyApproved(user, prisonCaseloadToDisplay, searchTerm)
+    ).map(c => {
+      return {
+        ...c,
```
Comment (gurps-singh) [inline server/routes/approvingLicences/handlers/approvalCases.ts:29]: **note:** ‏Testing in dev showed the date formatting was off - added in to format the dates as they appear in the caseload currently.

**Review by louisenorris — APPROVED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**